src/adam_bashforth_methods.jl

```diff
@@ -0,0 +1,647 @@
+################################################################################
+# Variable Step Variable Order (VSVO) Adam-Bashforth Implicit Solver
```

Comment:
The "Implicit" may be a bit missleading here, as it is not suitable for stiff problems.  Might be best to remove it?